data/members/gholad.yml

```diff
@@ -0,0 +1,6 @@
+id: gholad
+display-name: Gholad
+bio: I have interests in functional programming but please calm down, i'm really bad.
```

Comment:
I find the message unnecessarily negative, how about something like this:
```diff
- bio: I have interests in functional programming but please calm down, i'm really bad.
+ bio: I'm interested in, and discovering, functional programming
```

Reply:
its ok, thanks